assignments/objects.js

```diff
@@ -49,16 +90,37 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  name: "Susan",
+  age: 70,
+  speak: function() {
+    console.log("My name is Susan");
+  },
+  child: {
+    name: "George",
+    age: 50,
+    speak: function() {
+      console.log("My name is George");
+    },
+    grandchild: {
+      name: "Sam",
+      age: 30,
+      speak: function() {
+        console.log("My name is Sam");
+      }
+    }
+  }
+}
```

Comment:
While the directions say to use the 'this' keyword on this problem, I'm glad to see that you understand objects and nesting and calling functions nested deeply in objects. :heavy_check_mark: